scripts/pack_server.sh

```diff
@@ -106,6 +106,17 @@ copy_file `get_system_lib server aio` ${pack}/bin/`get_system_libname server aio
 copy_file `get_system_lib server zstd` ${pack}/bin/`get_system_libname server zstd`
 copy_file `get_system_lib server lz4` ${pack}/bin/`get_system_libname server lz4`
 
+DISTRIB_ID=$(cat /etc/*-release | grep DISTRIB_ID | awk -F'=' '{print $2}')
```

Comment:
what's the meaning of these code?

Reply:
Because binary on linux distribution based on debian requires dependencies of icudata, icuuc, icui18n. If I build a binary on Ubuntu18.04, and want to distribute it to other platforms, like linux in docker, I need to include the libraries in my package.

See this:  https://github.com/wkhtmltopdf/wkhtmltopdf/issues/2242